examples/kubectl.rs

```diff
@@ -118,10 +117,8 @@ impl App {
         if let Some(n) = &self.name {
             lp = lp.fields(&format!("metadata.name={}", n));
         }
-        let w = watcher(api, lp);
-
-        // present a dumb table for it for now. maybe drop the whole watch. kubectl does not do it anymore.
-        let mut stream = try_flatten_applied(w).boxed();
+        // present a dumb table for it for now. kubectl does not do this anymore.
+        let mut stream = watcher(api, lp).watch_applies().boxed();
```

Comment:
This can be [`pin_mut!`](https://docs.rs/futures/latest/futures/macro.pin_mut.html) rather than boxed

Reply:
yeah, i am using it elsewhere. i just don't want to mix pin into the easy go-to example.